infra/src/main/java/com/redhat/service/smartevents/infra/models/connectors/ConnectorType.java

```diff
@@ -8,7 +8,6 @@ public enum ConnectorType {
 
     final String value;
 
-    // We can not annotate the property `value` directly with `@JsonValue`. See https://issues.redhat.com/browse/MGDOBR-595
```

Comment:
I removed this comment because `ConnectorType` is actually not used in the API, thus it doesn't appear in `openapi.yaml`.